src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1208,6 +1208,38 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
             LowerHWIntrinsicFusedMultiplyAddScalar(node);
             break;
 
+        case NI_Sve_Abs:
```

Comment:
It would be nicer if we could do this via:
`if (HWIntrinsicInfo::IsEmbeddedMaskedOperation(intrinsicId)`.

That would prevent having to have a long list of `case NI_Sve_` statements which can be tricky to keep accurate.

Reply:
That's what we did for x64, we used flags and a common code path to simplify everything and avoid needing large amounts of special handling.

Reply:
We want to lower only those intrinsics that just has `predicated` version e.g. `Abs`. The intrinsics like `Add` that has both predicated and unpredicated shouldn't need to go through this. I can probably add a separate flag like `MaskedPredicatedOnly` or equivalent and then lower it here. Not sure how many of them are.

Reply:
I think a flag would be nicer, otherwise we'll need to have the more expensive jump table entries for a bunch of sparse intrinsic IDs